website/pages/_app.js

```diff
@@ -0,0 +1,18 @@
+import { ThemeProvider } from '@strapi/design-system/ThemeProvider';
+import { lightTheme } from '@strapi/design-system/themes';
+import PropTypes from 'prop-types';
+
+function MyApp({ Component, pageProps }) {
```

Comment:
Can you rename this component to `const App = ({ Component, pageProps }) => {` please and use an arrow function like every component in the repo 😄